src/analysis/processing/qgsalgorithmdeleteduplicategeometries.cpp

```diff
@@ -0,0 +1,199 @@
+/***************************************************************************
+                         qgsalgorithmdeleteduplicategeometries.cpp
+                         -----------------------------------------
+    begin                : December 2020
```

Comment:
December 2020? I don't think there's any reason to wait a year to merge this ;-P

Reply:
Australia is always in the future.